src/utils.ts

```diff
@@ -49,7 +50,7 @@ export function convertTreeToVisData(tree: any): TreeDate {
       id: parent.key,
       label: parent.label,
       description: parent.description,
-      nextStep: parent.nextStep,
```

Comment:
It confuses me. Shouldn't we save this code? It seems, we will lose nextStep in some situations